<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtrForm</class>
 <widget class="QWidget" name="AtrForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить среднюю скользящую</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditATR">
     <property name="inputMethodHints">
      <set>Qt::ImhPreferNumbers</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="maximumSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 255, 46);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Средняя скользящая</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="buttonLoad">
     <property name="text">
      <string>Загрузить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
